### DISCUSSION TITLE
Is it possible to display existing records as tags using the useSelect(),

### DISCUSSION THREAD

**Discussion by BenGardiner123** [refinedev/refine#1651] [Q&A] [answered]
I want to use the useSelect() to handle getting my data on edit, but my tags labels are an attribute from the object and then an attrbute from an object inside that object.
Is it possible to set the tags using what i would get from the useSelect?

![image](https://user-images.githubusercontent.com/61527372/158900932-2cc48288-dda9-4e0a-bdaf-f5a03dcd272c.png)

im currently pushing these tags in as children of the select
![image](https://user-images.githubusercontent.com/61527372/158901058-97b0209b-2213-4434-ae18-709fad9910ef.png)

![image](https://user-images.githubusercontent.com/61527372/158903147-6b2a8b01-bd36-4c32-b709-24851f801c41.png)

for example this is in the docs, but what if category had aa complex structure annd the label is to be one attribute from the top object and one attribute from the nested object

**Comment by omeraplak** [comment, marked as answer]
Hey @BenGardiner123 ,
Thank you for contacting us! The easiest way to do this is to use the `queryResult` returned from the `useSelect` hook.

I created a CodeSandbox for you, https://codesandbox.io/s/refine-use-select-example-forked-s55ne?file=/src/pages/posts/create.tsx

```jsx
const {
    selectProps,
    queryResult,
    defaultValueQueryResult
  } = useSelect();

  const allOptions = [
    ...(queryResult.data?.data || []),
    ...(defaultValueQueryResult.data?.data || [])
  ];

<Select
  {...categorySelectProps}
  options={allOptions.map((p) => ({
     label: `${p.title}-${p.id}`,
     value: p.id
   }))}
/>

```
